kickstarter/android-oss
# NTV-641: Formulary Screen, flag a project

# 📲 What

A project can be reported from the app

# 🛠 How
 Entire UI built in compose

# 👀 See
| Before 🐛 | After 🦋 |

https://user-images.githubusercontent.com/4083656/215604145-3912948b-203c-4451-97e7-14591ef235d9.mp4



| --- | --- |
|  |  |

# 📋 QA

- Go to a project, on the overview hit the "Report this project button", select one of the available categories, in the formulary screen write something on the input text and hit send button, you should:

1. See a progress bar below the toolbar
2. the email and url cannot be modified
3. once sent button has been pressed, the app will navigate back to the project overview, and the "report this project" button is no longer available, instead you have a text saying that you already reported the project

- Go to a project, on the overview hit the "Report this project button", select one of the available categories, in the formulary screen navigate back using a gesture, and the back button on the toolbar, you should be able to navigate back to the screen where the categories are selected.


# Story 📖

[NTV-642](https://kickstarter.atlassian.net/browse/NTV-642)


[NTV-642]: https://kickstarter.atlassian.net/browse/NTV-642?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
